p2p_wallet/Scenes/Main/Swap/ChooseSwapItem/ChooseSwapTokenCoordinator.swift

```diff
@@ -30,11 +30,11 @@ final class ChooseSwapTokenCoordinator: Coordinator<SwapToken?> {
             service: ChooseSwapTokenService(swapTokens: tokens, fromToken: fromToken),
             chosenToken: chosenWallet
         )
-        let view = ChooseItemView<ChooseSwapTokenItemView>(viewModel: viewModel) { [unowned self] model in
+        let view = ChooseItemView<ChooseSwapTokenItemView>(viewModel: viewModel) { [weak self] model in
             ChooseSwapTokenItemView(
                 token: model.item as! SwapToken,
                 chosen: model.isChosen,
-                fromToken: fromToken
+                fromToken: self?.fromToken ?? true
             )
```

Comment:
ViewBuilder does not let return nil value so we take default "true" param however screen is closed and it will not matter here as I suppose. It is a confusion between list render and close of the screen

Reply:
I added comment 
https://github.com/p2p-org/p2p-wallet-ios/pull/1192#discussion_r1177826444